OpenRA.Mods.Common/ServerTraits/LobbyCommands.cs

```diff
@@ -897,7 +897,9 @@ static Session.Slot MakeSlotFromPlayerReference(PlayerReference pr)
 		static void LoadMap(S server)
 		{
 			server.Map = server.ModData.MapCache[server.LobbyInfo.GlobalSettings.Map].Map;
-			server.LobbyInfo.Slots = server.Map.Players
+
+			server.MapPlayers = new MapPlayers(server.Map.PlayerDefinitions);
```

Comment:
Oh, I see.  Strange that my IDE didn't reveal this when I searched for references.